shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -627,4 +672,32 @@ - (void)scrollWheel:(NSEvent*)event {
   [self dispatchMouseEvent:event phase:kHover];
 }
 
+int MapFile(const char* inPathName, void** outDataPtr, size_t* outDataLength, bool executable) {
+  int outError;
+  int fileDescriptor;
+  struct stat statInfo;
+  outError = 0;
+  *outDataPtr = NULL;
+  *outDataLength = 0;
+
+  fileDescriptor = open(inPathName, O_RDONLY | (executable ? O_EXCL : 0), 0);
```

Comment:
This needs to be checked for `EINTR` and retried. Consider doing something similar to the the `FML_HANDLE_EINTR` macro here (possibly copy it to common for use there).

Reply:
Does the macro need to be moved to common or can it be imported directly from fml?